Features5/tests/features/newUserRegistration.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Register
+   As a user
+   I want to register
+   So that I can use the contact fox app
+
+   Background:
+       Given user has been navigated to registration page
+   
+   Scenario: New user registration with valid inputs
+       When user enters following valid credentials:
+         |name    |email                |password   |confirmPassword  |
+         |Susmita |susmita412@gmail.com |zxcvbnm    |zxcvbnm          |
+       And clicks "Register" button
+       Then user should see the homepage
+
+    Scenario Outline: When new user registers with invalid inputs
+       When user enters "<name>","<email>","<password>" and "<confirmPassword>"
```

Comment:
may be use `data table` instead :+1:

Reply:
-> Those are for different scenarios, do we make data table for different scenarios too?